functest/deposit_migration_test.go

```diff
@@ -19,43 +19,42 @@
 package functest
 
 import (
-	"fmt"
+	"encoding/base64"
 	"math/big"
 	"testing"
 
+	"github.com/insolar/insolar/logicrunner/builtin/foundation"
```

Comment:
You shouldn't use foundation in functests

Reply:
Done